src/telcameras_v2/migrations/0019_drop_old_views_20210209_1153.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+VIEWS = [
+    'cmsa_1h_count_view_v1',  # TODO: Check whether anybody still uses this view
```

Comment:
Je zet hier een TODO neer, maar hieronder droppen we hem

Reply:
Todo was voor mezelf als sanity check voordat ik hem merge. Inmiddels heb ik dat gecheckt en dit comment weggehaald.